docs/journey/day1/prerequisites/argocd.md

```diff
@@ -1,10 +1,9 @@
+### ArgoCD Installation - with operator (Recommended)
 
-### ArgoCD Installation - with operator( Recommended )
-
-Create argocd namespace to install the operator
+Create argocd namespace to install the operator:
 
 ```shell
-oc create namespace argocd
+$ oc create namespace argocd
```

Comment:
Command line without a dollar sign make it easy to be copy and pasted.

Reply:
Yeah, but it's a very widely used convention.

The OpenShift documentation https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_cli/getting-started-cli.html uses the same convention